CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 <!-- next version -->
 
-## v1.0.0-rc10/v0.76.0
+## v1.0.0-rc9/v0.76.0
```

Comment:
This line should probably be relabeled to `v1.0.0-rc9/v0.76.1`, but maybe after that release is created?